### PR TITLE
Do not send NotFoundError to sentry

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: eox_core/settings/aws.py [hunk 1]
```diff
@@ -120,7 +120,24 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
         settings.EOX_CORE_SENTRY_INTEGRATION_DSN
     )
     if sentry_sdk is not None and sentry_integration_dsn is not None:
+        def before_send(event, hint):
```
Comment (felipemontoya) [inline eox_core/settings/aws.py:123]: For me it makes sense to define this function in a different file.
And also to make the list of ignored errors a configurable value

**Comment by morenol** [comment]
@felipemontoya can you take a look?

File: eox_core/settings/common.py [hunk 2]
```diff
@@ -49,3 +49,7 @@ def plugin_settings(settings):
 
     # Sentry Integration
     settings.EOX_CORE_SENTRY_INTEGRATION_DSN = None
+    settings.EOX_CORE_SENTRY_IGNORED_ERRORS = [
+        'xmodule.exceptions.NotFoundError',
```
Comment (felipemontoya) [inline eox_core/settings/common.py:53]: Can we put this as an example on the sentry.py module docstring and remove it from here.

Then we can add it only for the SAAS on EOX_CORE_SENTRY_IGNORED_ERRORS

Comment (morenol) [reply eox_core/settings/common.py]: I added that to the README

**Pull request merged**